packages/polymath-issuer/src/pages/ticker/components/ReserveTickerForm.js

```diff
@@ -127,7 +127,7 @@ const formikEnhancer = withFormik({
   },
   handleSubmit: (values, { props }) => {
     const { dispatch } = props;
-
+    values.ticker = values.ticker.toUpperCase();
```

Comment:
Hey, just a comment here, we should avoid mutating objects like you are doing here. It can lead to unexpected behaviour: for example that's the reason why React has a `setState` function and we do `this.setState({ blabla }) and not `this.state = { blabla }`.
So here instead you should better do something like:
`const formattedValues = Object.assign({}, values, { ticker: ticker.toUpperCase() }` (or
`const formattedValues = { ...values, ticker: ticker.toUpperCase() }`)
This way you don't mutate `values` but create a new object `formattedValues`.

Reply:
Agreed